notebooks/222-vision-image-colorization/222-vision-image-colorization.ipynb

```diff
@@ -0,0 +1,495 @@
KxsAjdmFK5LPvFSY3aWoRolhKMSCgRTBSmp5WRa5BvCE2fFsA1YmtTsG1jv2OWJKzG6s2D1GP6b2valLlfcSKchzIDU2CD5fF1IDg49h9kfJqs5S5kPOBZmTWKtUOvys5V69QOmFjyA5D5tz8HV4Vqsm+phaveOCze8HkFVdfX78c05s+d1aZdXEUC+8+pr5IunVKh2JJ4mPrjLAm8zd99xCoBVfKm7ay6uzpDmuLJ/F89NEkeOtx04nySCBCFXlw+Fwpsiu1+tMVlghQVsddOhVT7p5OI33cu6J1ORjfj/mqqw+LseIlRt4jtww8X5wPHgcJl/pcdLzj3OQZK5pmnxfeD/5mb7vzzYYOhow903rDZEmjhw3kmVdlJvfM/clA8jzXp8vr0OrlySy+vv05kLPuXNv2qSQzjcKep7xR1/jU/dRb2aommqSOfdBc2fOezU/Hr+X56oV9U/t1SyWKBVZLQspY2TSeJpxEFIBIISIIYxSlifFOkPwGJhUVJSQ1pWpZFHfTUTK2uzHRMoiL1yRytfMWgtbSfRwLpHgQnqUG5eyi0OQpTGyL32FMoS0mI0fYHbOuo4RDqKYWWskZBqrhI8+Wxp07DBjdjo3nzKsmXTjEnkrXJG9l8zIbupmEk2S6me/A7NRlqnA/IhhEIsEu3AxackYAzcWWGfMnvzFMdLvl571GWaXVZUTsUjoDoc9DoedhOGHAYf2iM1qlRsdUMgwNokL6T95P5tayWavIbOzGRFPSWkMf5vEPA1JKiTEPD2bBZyNZ5iNGBGjT5hdSJ3U1SqTw6ZqEmZPVhQfQi4vRcwWP68QJ2vEN8t2wDHGVNB//ACzh77PPtK+T8+HIwby3N0TmC0qspRziwBLWoE2lypvxuwMs30IsJG5MlNGvTS6EOsB24wy8UqXess5F1wjrIVzFtaQU0lCVa4xqzbwIUU85FZJfdScKOmFSEbnYJzMe6T7btL3kFRLs4tUis5/iNlnm8ZQorc9hqHHOETEwMYcotKK3SLCBwtjRsQonc5YfeO7Xh8lq/pG65ObEx96R+f97fWCrMOrHEz+jqWLqCgtFgtsNtKiT7eiJNhoYsFzG0fpCqIXMR6PBJLha3opubAxFKtDmVxMeW58UPSPVtLKsjzzd1KCH4ZBWr7NVGe+qKTp7yVB5GfYiYjXOA8Xz++J9mPq4wDnPkeOcQghq8YcH44lFVfdJUqHrudqKM9Fb1D49/nG53Q6ZZuFnmdZ0UnnTZKVH35/Xs6I31lVVe6qpImwvk8hhDwv5oo95zxD+zo5TauSVA20j3ceOuNGzlrxEZNUWWvz+fG4VMiZ8MT5zMVFbybYmEB/nx53vanhter5rMku74dOxNMEmmF4Tdj5/1SYef38f32+fB44Vhoz5psZ/k7PF17XnHRrjNHv47/x+/Vz9qm86maZQ2rRWAwhwMcIG6dED5sWhbW1orR68VbajMvJimUMSpcauhgDEyIGL91wDoe9YHbquLRcLHGxuYAPHrvdVmG2JBr5IF2ymE0MYzCMI9qM2RQghIjFGDEOAw6HPXa7LU7tCcMHmC1lkibslDHYp3NzbmpWUFX1B5hdlCUOJ5mjSIrYOAgp9+M+kwMYqUGZtEgpE5Su2zqpRsBEGT4PTd1Ie9lxkEL5/kO8kvEZFWaPKaQ6QBLQ5BXTJgIx4mJzkQvUH44HnI6yptEqUFiDpqqwbBpcZ8xeTJidqib4RPjl2ZHQtHOQBKV0L3LnLMdIT4nj6Yjj6QQXJiuAMUYS0wAEhd3nmD1KuDwReJKxqqpwsbmYWqRmzE5eUyMe1KIoZ5gtZBemRq2qA7iUOBSTSn9q2aFw/ACzpSxTqkLhnGxcjIGxci+cY9UeUVIrhdnj0OOUMLvte5zMEX5cIS496rJEXVUoygoREDU3Y7ZcG4xDgDTHGNM9GcYB4RTET54xz6Tapx9iNtcEmTupSkffYTQ2E/YQpMUpS8b1fZevfxx6DH0vymvG7MOTmB2KIteuZcIZMZnvLfieKCWvbFLRs9IeDIwtpkQ8CppWnq+yrD6K2R8lq8+fP88Tj5NvrnCSuG232zOlRydRaMWECTj6oeXDqglI27YoyxIvXrzIizgnP4kVgLygaivAcrnMfju9mA/DkEtsXV1d5RuuFS/tBR2GAfv9/ozcUkEzxpwprdryQIIXgiSxsAUqO0BdX19nAqtJAM+BY8hFnfUs535Wfo9WHbuu+2Bhn98rkrbD4XBGFFarFa6vrzOh0UlU9Pnq8dQkmGM5V+L0NfJ7+IBdXFzg5cuXebypuDNUrv9OoOKY6kQrp0CTGyNevyZm/F4NpgQ5kih9fXrsWFWA36fV7gx+6vNd1+UxY9cv3ltda5ZEVntbdfUC+rp4LpeXlx88g1ppnt8PPcc5v3W5Mv4YY7I/Vau5Wt1nrWOO43zxnasRTdOcLVh6AdNziJGBuYqsoxx8nuj31YSXc0cnV5K4f2qv6xefAYhJufDw45DH1VkLUziUvkY1DHjcPogiBWnHWpfVWXtTeh+XzTJh9nj2TDHzfxgHBJ8abpQFXrz4DF3XppI7koks4fgDIpCSo4ZEJivpM79cSa3NugFSIwAfJTHpcbdFU9e4unqW5jOTegtYO23OpetQj90+bbj7XmwEfYWiaGEMktIqJLqq65zQIyH1Bkfv0Y0jtttHxBiwXq1xsb7As+trHA7Hc8z2I1wqBA8jiTQOAIzBsU2tt72U2TrH7IV0KkqksO1aSAg+EaEUQpb3T93/QvA47PdZgTbGYLNa4cWz5zlsXxUFqkqU1KZu4IqJVIaM2VPiT1mWKSFoKrMUAjen510NfRhxeXmJl599Pq2R45BC5W3ecGvMdmeY3aXavELIpuedjSwmRdcY8VsbIFc2YFtewewUmWQUJl2fdMByfynMZpibFoWua1HVNaqiwnq9wXqzmWE2GyxIgmGfyN6HmH3Ebr/L53JxeQmYIUcZpArChxVOiqJISYJy/84xezgrA9j3
```

Comment:
![image](https://user-images.githubusercontent.com/10940214/180785713-c53a0e62-019f-4629-bd0b-f7ac5757a50c.png)
 Please fix this error.